src/invidious/videos/parser.cr

```diff
@@ -55,7 +55,7 @@ def extract_video_info(video_id : String)
   client_config = YoutubeAPI::ClientConfig.new
 
   # Fetch data from the player endpoint
-  player_response = YoutubeAPI.player(video_id: video_id, params: "", client_config: client_config)
+  player_response = YoutubeAPI.player(video_id: video_id, params: "2AMB", client_config: client_config)
```

Comment:
Not sure if I should put `2AMB` in a static variable somewhere for using it in `try_fetch_streaming_data` too.